gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/PipelineController.groovy

```diff
@@ -90,6 +90,32 @@ class PipelineController {
     }
   }
 
+  @CompileDynamic
+  @ApiOperation(value = "Batch update pipeline definitions")
+  @RequestMapping(value = 'batchUpdate', method = RequestMethod.POST)
+  void batchUpdate(@RequestBody List<Map> pipelines) {
+    def operation = [
+      description: (String) "Batch updating pipelines",
+      application: pipelines.first().get('application'),
```

Comment:
I wonder if it's worth grouping these by application into separate `job`s. Even though the UI would always be sending in pipelines from one application, API users could send in updates to pipelines in several applications (which is what we do internally with that endpoint).

Reply:
@ajordens thoughts on this?

Reply:
Ah thank you for pointing that out -- just updated so that it groups the pipelines by application, then loops through the groups and kicks off a task for each set of pipelines -- it works, but I don't know anything about groovy so feel free to let me know if there's a better way you would have done it!